container/go/pkg/oci/reader_test.go

```diff
@@ -0,0 +1,123 @@
+package oci
+
+import (
+	"testing"
+
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/types"
+	"github.com/google/go-containerregistry/pkg/v1/validate"
+)
+
+var readertests = []struct {
+	// A descriptive name for this test case.
```

Comment:
The convention is to comment fields starting with the name of the field. So something like
`// name is the name of this test case.`

and the same for every other field below.